style.css

```
@@ -0,0 +1,179 @@
+@import url('https://fonts.googleapis.com/css2?family=Merriweather:wght@300&family=Roboto&display=swap');
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body, button {
+    font-family: Roboto, sans-serif;
+}
+
+.box {
+    flex-grow: 1;
+    position: relative;
+}
+
+p {
+    color: #0C0C0C;
+}
+h1,h2,h3 {
+    font-family: Merriweather, serif;
+}
+
+h1, h2 {
+    text-align: center;
+    color: white;
+}
+
+h1 {
+    flex-direction: column;
+    font-size: xx-large;
+}
+
+h2 {
+    color: #4845E4;
+    font-size:x-large;
+}
+
+.header {
+    background-color: #4845E4;
+    padding-block: 150px;
+    /*padding-bottom:150px;
+    padding-top: 150px;*/
+    flex-direction: column;
+    position: relative;
+}
+.inner-container {
+    flex-basis: 1200px;
+    flex-direction: column;
+}
+
+.squiggle-stripe {
+    position: absolute;
+    width: 50px;
+    height: 45px;
+    left: calc(50% - 40px);
+    justify-content: center;
+    align-content: flex-end;
+    bottom: 0;
+
+}
+
+nav ul {
+    list-style-type: none;
+    padding: 0;
+    display: flex;
+    top: 0;
+    right: 0;
+    position: absolute;
+
+}
+
+nav a {
+    color: white;
+    display: block;
+
+}
+
+nav li {
+}
+
+button {
+    background-color: #ED4385;
+    color: white;
+    text-transform: uppercase;
+    font-size: small;
+    border: 15px;
+    border-radius: 25px;
+    padding: 20px;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    margin-left: 650px;
+
+
+}
```

De margin-left zorgt ervoor dat die samen met flex niet zo netjes uitlijnt. Als de parent display: flex heeft én daarop de align-items heeft ingesteld, dan hoef je niet met een margin te werken om het in het midden te krijgen:
```suggestion
button {
    background-color: #ED4385;
    color: white;
    text-transform: uppercase;
    font-size: small;
    border: 15px;
    border-radius: 25px;
    padding: 20px;
    width: 200px;
}
```